### PR TITLE
#186 Add storybook for OpsModalButton wrapped in a table.

### PR DESCRIPTION
**Pull request by martinfrances107**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/235

## Testing instructions

run:-

yarn storybook

See new page for the OpsModalButton

The button is on the right.

### CODE REVIEW

File: src/components/02_atoms/OpsModalButton/OpsModalButton.stories.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import EditIcon from '@material-ui/icons/Edit';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import OpsModalButton from './OpsModalButton';
+
+storiesOf('Table Operations/OpsModalButton', module).addWithJSX(
```
Comment (lauriii) [inline src/components/02_atoms/OpsModalButton/OpsModalButton.stories.js]: Couldn't this component be used outside of tables? 🤔

Comment (martinfrances107) [reply src/components/02_atoms/OpsModalButton/OpsModalButton.stories.js]: > Couldn't this component be used outside of tables?

Maybe, 

The Ops in OpsModalButton - is short for 'object in the operations column of a table'.

I am still thinking of what are the next steps.... I could put the table as a decorator so when a user reads the bottom section of the OpsModalButton he/she is not immediately distracted.

Or maybe there would be two pages one showing the button in isolation and one with the button in the table? I will look at this some more and fix this up tonight.

**Comment by martinfrances107** [comment]
See below I had made the storybook easier to grasp, using the addDecorator() method the JSX tab just contains the OpsModalButton element.


hiddentable

**Comment by martinfrances107** [comment]
After a chat with lauriii....

I have split the representation of the button into two storybook pages.

1) The button in isolation.
2) The button contained in the operations column of a table.

**Pull request merged**